src/GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Tests/ProgramTests.cs

```diff
@@ -152,47 +154,47 @@ public void ShouldBeConfigured()
+                                                         "App": {
+                                                              "SearchStorageConnectionString": null,
+                                                              "SearchableDocumentsContainerName": "searchable-documents-container-name"
+                                                         }
+                                                      }
+                                                      """);
 
                 // ACT
-                var actual = Record.Exception(startupHost);
+                var host = StartupHost();
 
                 // ASSERT
-                Assert.NotNull(actual);
-                Assert.IsType<ConfigurationErrorsException>(actual);
+                var options = host.Services.GetRequiredService<IOptions<ContentApiOptions>>();
+                Assert.NotNull(options);
             }
             
             [Fact]
-            public void WhenContainerNameNotConfiguredShouldThrowOnStartup()
+            public void WhenContainerNameNotConfiguredShouldNotThrowOnStartup()
             {
                 // ARRANGE
-                Action startupHost = () => base.GetSutWithConfig(FullConfig,
-                    """
-                    {
-                       "App": {
-                            "SearchStorageConnectionString": "DefaultEndpointsProtocol=https;EndpointSuffix=core.windows.net;AccountName=mystorageaccount;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=https://mystorageaccount.blob.core.windows.net/;FileEndpoint=https://mystorageaccount.file.core.windows.net/;QueueEndpoint=https://mystorageaccount.queue.core.windows.net/;TableEndpoint=https://mystorageaccount.table.core.windows.net/",
-                            "SearchableDocumentsContainerName": null
-                       }
-                    }
-                    """);
+                IHost StartupHost() =>
+                    base.GetSutWithConfig(FullConfig, """
+                                                      {
+                                                         "App": {
+                                                              "SearchStorageConnectionString": "DefaultEndpointsProtocol=https;EndpointSuffix=core.windows.net;AccountName=mystorageaccount;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=https://mystorageaccount.blob.core.windows.net/;FileEndpoint=https://mystorageaccount.file.core.windows.net/;QueueEndpoint=https://mystorageaccount.queue.core.windows.net/;TableEndpoint=https://mystorageaccount.table.core.windows.net/",
+                                                              "SearchableDocumentsContainerName": null
+                                                         }
+                                                      }
+                                                      """);
```

Comment:
Just wondering if it might be better to use connection string `UseDevelopmentStorage=true`.

I know it's not an integration test and nothing is running but it's a shortcut to

```
DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;
AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;
BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;
QueueEndpoint=http://127.0.0.1:10001/devstoreaccount1;
TableEndpoint=http://127.0.0.1:10002/devstoreaccount1;
```

so it is a valid connection string and it avoids the long line here. Could also use this on L20, L139.